### ISSUE TITLE
GraphQL formatting of enums should not always put every item in a new line

### ISSUE THREAD

**Issue by koresar** [prettier/prettier#4811]
**Prettier 1.13.7**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuECCuBbABAYQm2AJwE8BJAZwgCYtgAdKLLAQQDEWANFgGSwBEAWiwDKLPiwDyLUiwCKLAOIsASiwCyLAOosAqiwAqLIQCEAElmOjj448YsBNCwFELJgFIWNxw8anH5xswW2sYAahaqpH4AchZKxuzGMgDS5jgaOEE44cmOOOz64ji8OLE4XDg4uFLJGUo4Rcm4qjgypi16ONo4eUJ8TXwefBp8Uk5VTkrC4QryTqpOLk6GrE2sUqwerDKsqgrsAAoF7ApBChoKLnwuCuYKMgp7vAriCgdYCnoKhgpKCrEKbQKRymQymDShIKmWKmJqmPSkUSkWKkcSkSLyUguUgaUi8UiGNwaNzvNwuNw1ITJFzJFLvZKqZLaZJKbhBbjhbh2biibiqbiObgybiGbh6NRSNRNNRKNTaNSONThNS8NSGNTmNTyNSqNR6ezqrisDRqcRqKpqWJqFxqURqIJqIRqDTRILRVTRd7RXjRKrRITRGTRFzRJTRPTRdhqL1SCQaA5NA7aA6iA5BA5KA6OA4uA7mA6xA68A6GA6qWRBZQuZRSZR6ZQhXjCczJWLcC2HCFVTSiYQaYSGYRBYSxZS9qrCJtTLjCJrCGTCOzCKQCM690ROXlz8TCVTCDzCITCFw4czCRz6bT6Dz6IT6cz6Xj6JT6Jr6KT6Qz6WL6VT6DT6FU+jhDoSg6EEzA3HYOiiDoGg6I4OhCKEeihC4oSxKESihDImjsE45j2C4AgaAImgMAAviAAA0IAQAADjAACW0DkMgoAAIaEIQEAAO4HFxCBsSgHEAG4QExAAmNEgAARoQHFgAA1nAMDCPRilMVAADmyAwIQaBwLRWnkHAhAwAcCnaRgHHIAAZhxAA2pm0QAVuQAAexgKcpqnCBxGBwNwWlwPZTkuSAGmEKZhDICA2kKfRAAWACOjkyfRhBaTAmhSTASXIAAHAADLRmUQKZmiJXFmVwDFomhbRhBwClaBMc1lkcdZtlIA5zlGSApkYExekGQN5BadpjlwLIaAQPAYX9bRMAcbJuWSflyBUMtClMY5k14BgNlxVA0CNSAaCmfoq3CX1pkURRQA)
```sh
--parser graphql
```

**Input**:

```graphql
enum CountryIso2 {
  AF AX AL DZ AS AD AO AI AQ AG AR AM AW AU AT AZ BH BS BD BB BY BE BZ BJ BM BT BO BQ BA BW BV BR IO BN BG BF BI KH CM CA CV KY CF TD CL CN CX CC CO KM CG CD CK CR CI HR CU CW CY CZ DK DJ DM DO EC EG SV GQ ER EE ET FK FO FJ FI FR GF PF TF GA GM GE DE GH GI GR GL GD GP GU GT GG GN GW GY HT HM VA HN HK HU IS IN ID IR IQ IE IM IL IT JM JP JE JO KZ KE KI KP KR KW KG LA LV LB LS LR LY LI LT LU MO MK MG MW MY MV ML MT MH MQ MR MU YT MX FM MD MC MN ME MS MA MZ MM NA NR NP NL NC NZ NI NE NG NU NF MP NO OM PK PW PS PA PG PY PE PH PN PL PT PR QA RE RO RU RW BL SH KN LC MF PM VC WS SM ST SA SN RS SC SL SG SX SK SI SB SO ZA GS SS ES LK SD SR SJ SZ SE CH SY TW TJ TZ TH TL TG TK TO TT TN TR TM TC TV UG UA AE GB US UM UY UZ VU VE VN VG VI WF EH YE ZM ZW
}
```

**Output**:

<details>
  <summary>Click to expand, it's 250 lines (which is the issue)</summary>

```graphql
enum CountryIso2 {
    AF
    AX
    AL
    DZ
    AS
    AD
    AO
    AI
    AQ
    AG
    AR
    AM
    AW
    AU
    AT
    AZ
    BH
    BS
    BD
    BB
    BY
    BE
    BZ
    BJ
    BM
    BT
    BO
    BQ
    BA
    BW
    BV
    BR
    IO
    BN
    BG
    BF
    BI
    KH
    CM
    CA
    CV
    KY
    CF
    TD
    CL
    CN
    CX
    CC
    CO
    KM
    CG
    CD
    CK
    CR
    CI
    HR
    CU
    CW
    CY
    CZ
    DK
    DJ
    DM
    DO
    EC
    EG
    SV
    GQ
    ER
    EE
    ET
    FK
    FO
    FJ
    FI
    FR
    GF
    PF
    TF
    GA
    GM
    GE
    DE
    GH
    GI
    GR
    GL
    GD
    GP
    GU
    GT
    GG
    GN
    GW
    GY
    HT
    HM
    VA
    HN
    HK
    HU
    IS
    IN
    ID
    IR
    IQ
    IE
    IM
    IL
    IT
    JM
    JP
    JE
    JO
    KZ
    KE
    KI
    KP
    KR
    KW
    KG
    LA
    LV
    LB
    LS
    LR
    LY
    LI
    LT
    LU
    MO
    MK
    MG
    MW
    MY
    MV
    ML
    MT
    MH
    MQ
    MR
    MU
    YT
    MX
    FM
    MD
    MC
    MN
    ME
    MS
    MA
    MZ
    MM
    NA
    NR
    NP
    NL
    NC
    NZ
    NI
    NE
    NG
    NU
    NF
    MP
    NO
    OM
    PK
    PW
    PS
    PA
    PG
    PY
    PE
    PH
    PN
    PL
    PT
    PR
    QA
    RE
    RO
    RU
    RW
    BL
    SH
    KN
    LC
    MF
    PM
    VC
    WS
    SM
    ST
    SA
    SN
    RS
    SC
    SL
    SG
    SX
    SK
    SI
    SB
    SO
    ZA
    GS
    SS
    ES
    LK
    SD
    SR
    SJ
    SZ
    SE
    CH
    SY
    TW
    TJ
    TZ
    TH
    TL
    TG
    TK
    TO
    TT
    TN
    TR
    TM
    TC
    TV
    UG
    UA
    AE
    GB
    US
    UM
    UY
    UZ
    VU
    VE
    VN
    VG
    VI
    WF
    EH
    YE
    ZM
    ZW
}
```
</details>

**Expected behavior**:

Anything but 250 lines. :) Maybe leave it within a single line, or make it 4 lines broken by 80 char limit.

Basically, any solution is good, but not the 250 lines. :)

Thanks!

**Comment by j-f1** [comment]
IMO this is one of the cases where `prettier-ignore` should be used:

**Prettier 1.13.7**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBiABABwE5xjASzmwFoCBzKCXAHSgQFcBbdAYQgdmwE8BJAZwgAmdMDrp0AQQBiUgBpSAMugAiALSkBlKSqkB5KbykBFKQHEpAJSkBZKQHUpAVSkAVKRoBCACXSftnrqenn4Amn4Aon5eAFJ+dp7ungaepp6Sfo6eAGp+1rwpAHJ+Fp6ynkYA0r6sdqwZrLmV4ayyrrqsyqzFrAqsrGwGlXUWrB2VbNasRt5TLqyOrC0aKhMqcSp2KgYRAxEWmrlmphHWEVER7tIT0gbScdJG0tZmsgAKbbJmGWZ2ZlEqKJmXxmIxmF7KMy6MxvdBmFxmdxmCxmYpmRxmcLedzeOzZDLeYreCbeFy8bS8Yq8XS8fKmXhRXh2XjKXjuGJ2GKwmJRGJDDSVKKVKqwyrWSqOSoWRQZRS5RQhRTaRTWRThRRGRTuRQuGwGGwTGwWGyOGzhGy5GzKGzuGy+GymGzWGwuUK2hTSOw2XQ2AY2Yo2KI2bQ2DI2DQ2OyFDKFayFWGFZSFAaFDSFIyFKKFCyFFyFWQ2BMGPR2N4TN6ON7aN4ZN4WN7hN5RN6+N7FN7KN7uN7WYwZSxRSwGSwuSxZZSaXyVYqKP3vPEDezaTR2TTuTQZTTFSzLgaaCcHBSaCaaIyaEKaAxqH7L7QRZUn3SaayaOKaDSaKKsXyacKuRyuHErgaK4viuMorgWK4EyuAYrjuK4xSuNYrh2K4AyuLkTgWE4GSSECIRONoTh2E44ROBo2QuNkUTZMU2QWNkRj2LIES+KEURqHYaj2HQAC+IAADQgBAmCENA-DIKAACG2DYBAADubyyQgkkoNJABuEAEAAJkJIAAEbYNJYAANZ4JomAmQQUDkMgMDYAwcDCTZ-DEDAbzGeQTDScgABm0kADZucJABW-AAB6eMZZkWdJTBwIoNlwP5QUhSAVnYG52DICA5DGZgAAWACOgX6TgNkwPYukwIVyAABwAAzCTgEBufYBW5TgcDZRpKXCbgxUMAQuCedJ3m+UgAXBc5IBuUwBD2Y5s38DZ5CBXAxgMBA8CpTNwkwNJBnVTptXIEIB3GQQgVrewTA+blVD0PpDBua4R1qdNbl8XxQA)
```sh
--parser graphql
```

**Input:**
```graphql
# prettier-ignore
enum CountryIso2 {
  AF AX AL DZ AS AD AO AI AQ AG AR AM AW AU AT AZ BH BS BD BB BY BE BZ BJ BM BT BO BQ BA BW BV BR IO BN BG BF BI KH CM CA CV KY CF TD CL CN CX CC CO KM CG CD CK CR CI HR CU CW CY CZ DK DJ DM DO EC EG SV GQ ER EE ET FK FO FJ FI FR GF PF TF GA GM GE DE GH GI GR GL GD GP GU GT GG GN GW GY HT HM VA HN HK HU IS IN ID IR IQ IE IM IL IT JM JP JE JO KZ KE KI KP KR KW KG LA LV LB LS LR LY LI LT LU MO MK MG MW MY MV ML MT MH MQ MR MU YT MX FM MD MC MN ME MS MA MZ MM NA NR NP NL NC NZ NI NE NG NU NF MP NO OM PK PW PS PA PG PY PE PH PN PL PT PR QA RE RO RU RW BL SH KN LC MF PM VC WS SM ST SA SN RS SC SL SG SX SK SI SB SO ZA GS SS ES LK SD SR SJ SZ SE CH SY TW TJ TZ TH TL TG TK TO TT TN TR TM TC TV UG UA AE GB US UM UY UZ VU VE VN VG VI WF EH YE ZM ZW
}
```

**Output:**
```graphql
# prettier-ignore
enum CountryIso2 {
  AF AX AL DZ AS AD AO AI AQ AG AR AM AW AU AT AZ BH BS BD BB BY BE BZ BJ BM BT BO BQ BA BW BV BR IO BN BG BF BI KH CM CA CV KY CF TD CL CN CX CC CO KM CG CD CK CR CI HR CU CW CY CZ DK DJ DM DO EC EG SV GQ ER EE ET FK FO FJ FI FR GF PF TF GA GM GE DE GH GI GR GL GD GP GU GT GG GN GW GY HT HM VA HN HK HU IS IN ID IR IQ IE IM IL IT JM JP JE JO KZ KE KI KP KR KW KG LA LV LB LS LR LY LI LT LU MO MK MG MW MY MV ML MT MH MQ MR MU YT MX FM MD MC MN ME MS MA MZ MM NA NR NP NL NC NZ NI NE NG NU NF MP NO OM PK PW PS PA PG PY PE PH PN PL PT PR QA RE RO RU RW BL SH KN LC MF PM VC WS SM ST SA SN RS SC SL SG SX SK SI SB SO ZA GS SS ES LK SD SR SJ SZ SE CH SY TW TJ TZ TH TL TG TK TO TT TN TR TM TC TV UG UA AE GB US UM UY UZ VU VE VN VG VI WF EH YE ZM ZW
}

```

One could argue that the inlined enum is *less* readable since you have to scroll horizontally, but in this case you know you won’t need to read or update this list very often, so it’s reasonable to override Prettier here.

**Comment by koresar** [comment]
Oh! TIL "prettier-ignore"!
Thank you very much.

PS: My editor is doing visual "soft wraps" for long lines.
